### PR TITLE
MAT-1950 Create Timing Attribute Type Widget

### PR DESCRIPTION
**Pull request by ghost**
Add the following coding systems: EventTiming, TimingEvent, DaysOfWeek

Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/measure-loader/value_set_helpers.rb [hunk 1]
```diff
@@ -83,91 +83,96 @@ def code_systems_by_name()
...
+            'mediaType' => 'http://hl7.org/fhir/v3/MediaType',
+            'Diagnosis Role' => 'http://terminology.hl7.org/CodeSystem/diagnosis-role',
+            'DiagnosisRole' => 'http://terminology.hl7.org/CodeSystem/diagnosis-role',
+            'CDCREC' => 'http://cts.nlm.nih.gov/fhir/cs/cdcrec',
+            'HCPCSReleaseCodeSets' => 'http://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets',
+            'HCPCS' => 'http://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets',
+            'v3.GTSAbbreviation' => 'http://terminology.hl7.org/ValueSet/v3-GTSAbbreviation',
+            'EventTiming' => 'http://hl7.org/fhir/event-timing',
+            'v3.TimingEvent' => 'http://terminology.hl7.org/CodeSystem/v3-TimingEvent',
+            'TimingEvent' => 'ttp://terminology.hl7.org/CodeSystem/v3-TimingEvent',
```
Comment (adongare) [inline lib/measure-loader/value_set_helpers.rb:174]: probably typo: h**ttp**

**Review by adongare — DISMISSED** [review]

**Review by AndrewBird81 — DISMISSED** [review]

File: lib/measure-loader/value_set_helpers.rb (hunk 1 above)
Comment (serhii-ilin) [reply lib/measure-loader/value_set_helpers.rb:174]: good catch

**Review by adongare — APPROVED** [review]

**Review by AndrewBird81 — APPROVED** [review]
